css-canvas-medical/index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Canvas Medical</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row top-row bottom-border">
+        <div class="column-half">
+          <div class="blue-logo-container">
+            <img src="./images/logo-blue.svg" alt="" />
+          </div>
+        </div>
+        <div class="column-half buttons">
+          <div class="buttons-container">
+            <button class="white-button">Product</button>
+            <button class="white-button">Implementation</button>
+            <button class="white-button">About Us</button>
+            <button class="white-button">Blog</button>
+            <button class="blue-button">Contact Us</button>
+          </div>
+        </div>
+      </div>
+
+      <div class="row hero-container">
+        <div class="column-two-thirds">
+          <h1 class="no-margin-bottom tiempos-font">
+            The EMR Reimagined for Primary Care
+          </h1>
+          <p>
+            Can't remember the last time you talked with a patient without
+            focusing on the EMR? Canvas lets you rediscover the joy in the work
+            you do. Capture meaningful data, identify caps in care, and move
+            seamlessly between tasks within the platform. You stay present with
```

Comment:
You want to avoid using `'` in text because this denotes strings in TypeScript. You should use HTML escape entities like `&apos;` to denote an apostrophe.